package.json

```diff
@@ -70,6 +70,7 @@
   "files": [
     "domains.js",
     "domains.json",
+    "index.d.ts",
```

Comment:
According to TypeScript docs, there's no absolute need to add a `"types"` field to `package.json`:

> Roughly, the resolution will first check the optional `"types"` field, then the `"main"` field, and finally will try `index.d.ts` in the root.

https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson